hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/mover/Mover.java

```diff
@@ -649,6 +664,11 @@ static int run(Map<URI, List<Path>> namenodes, Configuration conf)
     Map<Long, Set<DatanodeInfo>> excludedPinnedBlocks = new HashMap<>();
     LOG.info("namenodes = " + namenodes);
 
+    DefaultMetricsSystem.initialize("Mover");
```

Comment:
Does Balancer have a similar metrics? If not shall we use this PR or create a new one to do that?

Reply:
We have it in the internal branch with similar implementation but it is not in upstream.

Surely, we can put the changes together. Let me update the JIRA to include the balancer metrics.